src/main/java/org/graylog/plugin/filter/dns/DnsResolverFilterMetaData.java

```diff
@@ -49,7 +49,7 @@ public URI getURL() {
 
     @Override
     public Version getVersion() {
-        return new Version(1, 2, 0, "SNAPSHOT");
+        return new Version(1, 1, 3, "nt1");
```

Comment:
Please fetch the version number from a properties file generated by the Maven build.

Example:
* https://github.com/graylog-labs/graylog-plugin-internal-logs/blob/2.4.0-beta.2/src/main/java/org/graylog/plugins/internallogs/InternalLogsInputPluginMetaData.java
* https://github.com/graylog-labs/graylog-plugin-internal-logs/blob/2.4.0-beta.2/src/main/resources/org.graylog.plugins.graylog-plugin-internal-logs/graylog-plugin.properties
* https://github.com/graylog-labs/graylog-plugin-internal-logs/blob/2.4.0-beta.2/pom.xml#L91-L101